src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -592,6 +652,8 @@ bool decompress(
             success = decompressImpl<8, true>(source, dest, source_size, dest_size);
         if (best_variant == 3)
             success = decompressImpl<32, false>(source, dest, source_size, dest_size);
+        if (best_variant == 4)
```

Comment:
We should activate this method with dynamic dispatching (checking cpuid at runtime).

Reply:
Done. Please help to review.